.github/workflows/ci.yml

```diff
@@ -40,3 +39,13 @@ jobs:
       env:
         COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         COVERAGE_VERSION: 1.19
+  test32bit:
+    runs-on: ubuntu-latest
```

Comment:
GitHub Actions doesn't provide 32-bit runners so the most reliable way to run as 32-bit is to use QEMU and Docker.

But, if this can be reproduced on a 64-bit runner simply running with GOARCH=386 that'd be fine. I'll mess with that and see if it works.

Reply:
ok, it was a bit more than setting `GOARCH` - I needed to tell `setup-go` to install the 386 version, but it works:

```
--- FAIL: TestChmod (0.00s)
panic: unaligned 64-bit atomic operation [recovered]
	panic: unaligned 64-bit atomic operation
```

Reply:
Very interesting, I wonder why the env var was insufficient :thinking: On my Linux machine it panicked as expected.